### ISSUE TITLE
S5 Mini Duos USB not working

### ISSUE THREAD

**Issue by vicenteicc2008** [msm8916-mainline/lk2nd#281]
my computer not detecting my s5 mini with lk2nd

**Comment by vicenteicc2008** [comment]
is msm8226

**Comment by wonderfulShrineMaidenOfParadise** [comment]
You might need to add a node, `samsung,muic-reset`.
https://github.com/msm8916-mainline/lk2nd/blob/master/dts/msm8916/msm8916-samsung-cprime.dtsi#L1-L4

Check your MUIC type and GPIO pins in downstream dts and change the values accordingly.

**Comment by vicenteicc2008** [comment]
muic-reset is not in the dts

**Comment by wonderfulShrineMaidenOfParadise** [comment]
Uh, so, add it then?

**Comment by vicenteicc2008** [comment]
but in the downstream it is not

**Comment by wonderfulShrineMaidenOfParadise** [comment]
> Check your MUIC type and GPIO pins in downstream dts

No. You don't look for it in downstream. You are looking for `MUIC type and GPIO pins`

**Comment by vicenteicc2008** [comment]
but how do i find it?

**Comment by wonderfulShrineMaidenOfParadise** [comment]
Ctrl + F and look for `sm5502` or `sm5504`

**Comment by vicenteicc2008** [comment]
i found it

**Comment by vicenteicc2008** [comment]
How can I know the pins and the address?

**Comment by wonderfulShrineMaidenOfParadise** [comment]
Look for keywords, `gpio` and `reg`.

**Comment by vicenteicc2008** [comment]
same problem

**Comment by wonderfulShrineMaidenOfParadise** [comment]
I am afraid that I don't understand your problem without codes pushed to GitHub.

**Comment by wonderfulShrineMaidenOfParadise** [comment]
Consider joining [#lk2nd:postmarketos.org](https://app.element.io/#/room/#lk2nd:postmarketos.org) for helps in chats.
